src/app/base/components/comment/comment.component.ts

```diff
@@ -21,13 +39,16 @@ export class CommentComponent implements OnInit, OnDestroy {
   }
   commentIdOptions: any[];
   allOpen: boolean = false;
+  isSlideOverOpen: boolean = false;
+  positionComment: string = 'right';
 
   constructor() { }
   ngOnDestroy(): void {
   }
   ngOnInit(): void {
     this.initDataManger();
     UserManager.registerAfterInitActionOrRun(this, () => this.initUsers(), true);
+    localStorage.setItem('commentPosition', this.positionComment);
```

Comment:
i think this isn't what we want (on init I would assume we need to load it from the local storage and then set it to the variable if already present)